added the link to solution of Project-2

ChaturvediSulabh opened the pull request:
This is a PR template. If you are adding a solution link to the curriculum, leave this as is. If not, delete it and write the message you wish.

Thank you,

The Odin Project team.

105ron commented:
Hi @ChaturvediSulabh,
This repository holds the files that make up our website. We require you to add a link to your solution, not the actual solution. If you need help with how to do this, head on over to the general room on our Discord chat room. Thank you.

ChaturvediSulabh commented on ruby_programming/basic_ruby/project_building_blocks.md:
```diff
@@ -18,1316 +19,1321 @@ There's a video about it [from Harvard's CS50 class](https://www.youtube.com/wat
+- [Kevin Mulhern's solution](https://github.com/KevinMulhern/ruby_building_blocks/blob/master/caesar_cipher.rb)
+- [Emil Johansen's solution](https://github.com/EmilOJ/odin_project/blob/master/Ruby_Building_Blocks/caesar_cipher.rb)
+- [Sasikala's solution](https://github.com/Sasikala-Ravichandran/my_ruby_projects/blob/master/caesar_cipher.rb)
+- [Sam Padrul's solution](https://github.com/sampadrul/Ruby-Projects/blob/master/caesar%20cipher/caesa
```
I have added the link to my solution

ChaturvediSulabh commented on ruby_programming/basic_ruby/project_building_blocks.md:
```diff
@@ -18,1316 +19,1321 @@ There's a video about it [from Harvard's CS50 class](https://www.youtube.com/wat
+- Add your solution below this line!
+- [Awais Ali Khan](https://github.com/awaisalikhan/ruby_programming/blob/master/caesar_cipher.rb)
+- [Mike Farias' Solution](https://github.com/mkezmkez/ruby-buildingblocks/blob/master/practice1.rb.arb)
+- [Sulabh Chaturvedi's Solution](https://github.com/ChaturvediSulabh/the-odin-project/blob/master/Building_Blocks/project1/caeser_cipher.rb)
```
this is the link to my solution

ChaturvediSulabh commented:
> Hi @ChaturvediSulabh,
> This repository holds the files that make up our website. We require you to add a link to your solution, not the actual solution. If you need help with how to do this, head on over to the general room on our Discord chat room. Thank you.

I apologize for the misleading PR title, I have added the link to my solution and not the solution 

> Hi @ChaturvediSulabh,
> This repository holds the files that make up our website. We require you to add a link to your solution, not the actual solution. If you need help with how to do this, head on over to the general room on our Discord chat room. Thank you.

apology for the misleading pr title. though, I've added  [my solution link](https://github.com/TheOdinProject/curriculum/pull/13173/files/e861d689ae22541bea3c24bec1fc0d19d956e294#r347106581) and not the actual solution.

105ron commented:
Ok, it looks like you have an auto formatter enabled. There's  2,532 changes in the file and that is too many for me to review in a single PR. Could you disable it and only change the line you need to add your solution?
